src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,108 @@
+package duke.ui;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Interface that reads in inputs and displays outputs.
+ */
+public abstract class Ui {
+
+    protected Scanner sc;
+    protected String msg;
+
+    /**
+     * Constructor for Ui.
+     *
+     * @param sc The scanner from which inputs are read.
+     */
+    public Ui(Scanner sc) {
+        this.sc = sc;
+        this.msg = "";
+    }
+
+    /**
+     * Method to set display message.
```

Comment:
Shouldn't the comment start with a verb?

Reply:
Right, thank you!